azurerm/internal/services/web/resource_arm_function_app.go

```diff
@@ -661,16 +658,11 @@ func getBasicFunctionAppAppSettings(d *schema.ResourceData, appServiceTier strin
 		})
 	}
 
-	consumptionSettings := []web.NameValuePair{
-		{Name: &contentSharePropName, Value: &contentShare},
-		{Name: &contentFileConnStringPropName, Value: &storageConnection},
-	}
-
 	// If the application plan is NOT dynamic (consumption plan), we do NOT want to include WEBSITE_CONTENT components
 	if !strings.EqualFold(appServiceTier, "dynamic") {
 		return basicSettings
 	}
-	return append(basicSettings, consumptionSettings...)
+	return basicSettings
```

Comment:
If this change is intention, then maybe this should be just `return basicSettings` since the if and else are doing the same thing.

Reply:
changed this code logic. remove unnecessary code.